frontend/projects/ui/src/app/modals/action-marketplace/action-marketplace.component.html

```diff
@@ -0,0 +1,39 @@
+<ion-header>
+  <ion-toolbar>
+    <ion-buttons slot="start">
+      <ion-button (click)="dismiss()" class="enter-click">
```

Comment:
No `enter-click` on close buttons. This would be unique int he app and therefore be unexpected.